src/components/signMessage/signMessage.css

```diff
@@ -0,0 +1,49 @@
+@import '../app/variables.css';
+
+:root {
+  --padding-top: 40px;
+}
+
+.signMessage > div {
```

Comment:
I would prefer another class for the `div` instead of nesting with `>`. It makes it easier to see where the styles come from in the JSX